cypress/e2e/cloud/operator.test.ts

```diff
@@ -9,6 +9,7 @@ describe('Operator Page', () => {
           }).then(() => {
             cy.quartzProvision({
               isOperator: true,
+              operatorRole: 'read-write',
```

Comment:
should we add a basic test for when the operatorRole isn't `read-write` so that we know that certain functionality should be read only?